### ISSUE TITLE
One funder two publishers

### ISSUE THREAD

**Issue by KDuerden** [ThreeSixtyGiving/grantnav#768]
**Describe the bug**
Two central government departments have now published data about government grants.

https://grantnav.threesixtygiving.org/publisher/360G-cabinetoffice
https://grantnav.threesixtygiving.org/publisher/360G-DCMS

The funder pages only link to the Cabinet Office publisher page. For example:
https://grantnav.threesixtygiving.org/funder/GB-GOR-D17
https://grantnav.threesixtygiving.org/funder/GB-GOR-D18
https://grantnav.threesixtygiving.org/funder/GB-GOR-D4

This means that, unless you know the publisher prefix of DCMS, and that they funded, there isn't a link to the details about their data.

**Expected behavior**
This is an edge case, so I don't have any expectations, just flagging it.

**Additional context**
There have been other related issues around files have multiple hosting pages but the publisher page only allowing for one, and the issue of files with multiple Funders not formatting the names that nicely. When looking at the design of the Funder and Recipient pages, it could be worth looking at the Publisher pages too.

**Comment by KDuerden** [comment]
Wanted to update that individual grant records point to the relevant publisher pages, so it's only an issue with the Funder view & links.

**Comment by BibianaC** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [One funder two publishers](https://trello.com/c/leFCImWx/187-one-funder-two-publishers)

**Comment by mariongalley** [comment]
@KDuerden I don't think we have any data that fits this edge case anymore to test it - but I'd expect org pages to have solved this.

Happy to close?

**Comment by KDuerden** [comment]
> but I'd expect org pages to have solved this.

I'm not sure if that is the case. I'll need to check/remind myself how org pages prefix mapping works to confirm if it would automatically know what to do in this case.
